lib/dgram.js

```diff
@@ -121,6 +122,12 @@ function Socket(type, listener) {
 
   // If true - UV_UDP_REUSEADDR flag will be set
   this._reuseAddr = options && options.reuseAddr;
+  
+  if (options && options.recvBufferSize)
```

Comment:
If we have to store these, I'd say use symbols. It's kind of ugly though because if the values are updated later, these values aren't kept in sync. And, libuv provides APIs to get those values.